packages/explorer-2.0/components/Orchestrators/PerformanceTable.tsx

```diff
@@ -0,0 +1,539 @@
+    })
+  }
+
+  // Let the table remove the filter if the string is empty
+  fuzzyTextFilterFn.autoRemove = (val) => !val
+
+  function DefaultColumnFilter({ column: { filterValue, setFilter } }) {
+    return (
+      <Flex
+        sx={{
+          alignItems: 'center',
+          pl: 3,
+        }}
+      >
+        <Search sx={{ width: 16, height: 16, mr: 1, color: 'muted' }} />
+        <Box
+          value={filterValue || ''}
+          onChange={(e) => {
+            setFilter(e.target.value || undefined)
+          }}
+          placeholder={`Filter`}
+          as="input"
+          type="text"
+          variant="input"
+          sx={{
+            display: 'block',
+            outline: 'none',
+            width: '100%',
+            appearance: 'none',
+            fontSize: 2,
+            lineHeight: 'inherit',
+            border: 0,
+            color: 'inherit',
+            bg: 'transparent',
+          }}
+        />
+      </Flex>
+    )
+  }
+
+  const columns: any = useMemo(
+    () => [
+      {
+        Header: '#',
+        accessor: 'rank',
+        disableSortBy: true,
+      },
+      {
+        Header: 'Orchestrator',
+        accessor: 'id',
+        filter: 'fuzzyText',
+        Filter: DefaultColumnFilter,
+        mobile: true,
+        sortType: (rowA, rowB, columnID) => {
+          let a = getRowValueByColumnID(rowA, columnID)
+          let b = getRowValueByColumnID(rowB, columnID)
+          let aThreeBoxSpace = getRowValueByColumnID(rowA, 'threeBoxSpace')
+          let bThreeBoxSpace = getRowValueByColumnID(rowB, 'threeBoxSpace')
+          let rowAIdentity = aThreeBoxSpace?.name ? aThreeBoxSpace?.name : a
+          let rowBIdentity = bThreeBoxSpace?.name ? bThreeBoxSpace?.name : b
+
+          return compareBasic(rowAIdentity, rowBIdentity)
+        },
+      },
+      {
+        Header: 'Activation Round',
+        accessor: 'activationRound',
+      },
+      {
+        Header: 'Deactivation Round',
+        accessor: 'deactivationRound',
+      },
+      {
+        Header: 'ThreeBoxSpace',
+        accessor: 'threeBoxSpace',
+        filter: 'fuzzyText',
+        Filter: DefaultColumnFilter,
+      },
+      {
+        Header: 'Delegator',
+        accessor: 'delegator',
+      },
+      {
+        Header: 'Total Score',
+        accessor: `scores.${region}`,
+        mobile: true,
+        sortDescFirst: true,
+        defaultCanSort: true,
+        sortType: 'basic',
+      },
+      {
+        Header: 'Success Rate',
+        accessor: `successRates.${region}`,
+        mobile: false,
+        sortType: 'basic',
+      },
+      {
+        Header: 'Latency Score',
+        accessor: `roundTripScores.${region}`,
+        sortType: 'basic',
+        mobile: false,
+      },
+    ],
+    [region],
+  )
+
+  function getRowValueByColumnID(row, columnID) {
+    return row.values[columnID]
+  }
+
+  function compareBasic(a, b) {
+    return a === b ? 0 : a > b ? 1 : -1
+  }
```

Comment:
```suggestion
  function compareBasic(a, b) {
   a = a || 0
   b = b || 0
    return a === b ? 0 : a > b ? 1 : -1
  }
```

I think this should fix inconsistent sorting